site/content/en/docs/Concepts/principles.en.md

```diff
@@ -5,14 +5,13 @@ date: 2019-06-18T15:31:58+08:00
 
 The primary goal of minikube is to make it simple to run Kubernetes locally, for day-to-day development workflows and learning purposes. Here are the guiding principles for minikube, in rough priority order:
```

Comment:
We're seeing increasing confusion about what "locally" really means. There is a historic context, in where it meant "something running on your laptop". But nowadays we have both the people expecting the control-plane components to run directly on their development host, as well as the people having their desktop environment in a private cloud. This sentence is great for a slogan ("`Run Kubernetes locally`"), but could actually need some expanding elsewhere ?

From #4730

Reply:
What do you suggest for wording here? 

I believe our primary goal is to run it locally, as in "something on your laptop". Running elsewhere seems secondary.

Reply:
We should probably should stick with the "run Kubernetes locally". I would like it if it could include having a second computer right next to your usual one, or even something like private cloud - but agree those are not the primary goal.

Nothing wrong with the paragraph in general, though.